app/models/gamification_leaderboard.rb

```diff
@@ -1,13 +1,18 @@
 # frozen_string_literal: true
 
 class DiscourseGamification::GamificationLeaderboard < ::ActiveRecord::Base
+
+  # Needs to be updated alongside constant in gamification-leaderboard.js
+  USER_LIMIT = 100
```

Comment:
Where is this used?

Reply:
https://github.com/discourse/discourse-gamification/pull/20/files#diff-627c3f32b084dd3a565c35d1d4147c13d8b18f33aafc1b95e488c311f5ffce12R26